packages/zcli-core/src/lib/secureStore.ts

```diff
@@ -44,15 +44,15 @@ export default class SecureStore {
     return this.keytar
   }
 
-  setPassword (account: string, password: string) {
-    return this.keytar?.setPassword(this.serviceName, account, password)
+  setSecret (account: string, secret: string) {
+    return this.keytar?.setPassword(this.serviceName, account, secret)
```

Comment:
Do we need to change this `this.keytar?.setPassword` to `this.keytar?.setSecret` as well? Or maybe it cannot be changed because it's from a library?

Reply:
Unfortunately we can't, this interface is required by the `keytar` library which is a bit annoying :(